### ISSUE TITLE
install failed in Laravel 6

### ISSUE THREAD

**Issue by shaunthegeek** [webcraft/laravel-random#2]
error:

```
composer require webcraft/laravel-random
Using version ^2.0 for webcraft/laravel-random
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Conclusion: remove laravel/framework v6.2.0
    - Conclusion: don't install laravel/framework v6.2.0
    - webcraft/laravel-random 2.0.0.x-dev requires illuminate/support ^5.1 -> satisfiable by illuminate/support[5.1.x-dev, 5.2.x-dev, 5.3.x-dev, 5.4.x-dev, 5.5.x-dev, 5.6.x-dev, 5.7.17, 5.7.18, 5.7.19, 5.7.x-dev, 5.8.x-dev
```

**Comment by webcraft** [comment]
Thanks. Feel free to make a PR. Otherwise I will try to look into it in the next couple of days.

**Comment by webcraft** [comment]
Closed by #3
